Vizzuality/half-earth-v3
# Feature/promotional modal

## Release notes modal
### Description
This PR activates a modal with release notes on user first visit after some release.
A user config object is added to the local storage in order to avoid displaying the modal in any subsequent visit until another update is published.
The tutorial state has been also integrated on this local storage user config object in order to remove the dedicated store for the tutorial and to have all user config related keys on the same place.
Some utility functions have been created in order to ease the retrieval and addition of elements to the proper storage key.

![image](https://user-images.githubusercontent.com/6906348/95471666-79df2e00-0982-11eb-8bcc-3e2f11d80763.png)

**known issues**: Locally the custom styles for the modal are overridden by the he-modal styles from he-components. This is not happening in the `build` version of the app. You can check that by visiting the vercel preview.

### Designs
https://projects.invisionapp.com/d/main/#/console/17484384/433686278/preview
Text for the modal have been updated taking into account[ this piece of feedback](https://basecamp.com/1756858/projects/13899003/messages/91137471#comment_783360740)
### Testing instructions
Open http://localhost:3000/dataGlobe on a private window to see the modal. Check `application > local storage` to see the updates
### Feature relevant tickets
https://half-earth-map.atlassian.net/browse/HE-194
